symplyphysics/laws/gravity/gravity_force_from_mass.py

```diff
@@ -0,0 +1,28 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+from sympy.physics.units import gravitational_constant
+
+# Description
+## Field strength of gravity field E = G * M / R^2, where
```

Comment:
We already have this law defined, as `acceleration_free_fall`. Use it directly or derive from it, or derive it from more generic law.